.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1325,6 +1345,39 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             return ret;
         }
 
+        // Parse entities that are made up by two time points with now
+        private List<DateTimeParseResult> GetPrWithNow(MatchCollection matches, ExtractResult er, DateObject referenceDate)
+        {
+            var pr = new List<DateTimeParseResult>();
+            pr.Add(this.config.DateParser.Parse(er, referenceDate));

```

Comment:
This method should be only for the "now" code. The other entity (the date result) should not be extracted here.
Also, why call Parse() and not Extract like in the original code?